src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -4,23 +4,6 @@ import { Homepage } from './HomepageObjClass'
 provideEditingConfig(Homepage, {
   title: 'Homepage',
   attributes: {
-    cartAddedMessage: {
-      title: 'Confirmation message after adding',
-    },
-    cartAddLabel: {
-      title: 'Add-button label',
-    },
-    cartRemovedMessage: {
-      title: 'Confirmation message after removing',
-    },
-    cartRemoveLabel: {
-      title: 'Remove-button label',
-    },
-    cartUnvailableMessage: {
-      title: 'Add-button tooltip if not logged in',
-      description:
-        'The placeholder __product__ is available for all labels and messages.',
-    },
```

Comment:
Please provide a working copy where you cleared these values.

Reply:
I just did: https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-localized-product-area.scrivito-portal-app.pages.dev/de?_scrivito_workspace_id=g3067a03b4ddcd12&_scrivito_display_mode=view